README.md

```diff
@@ -40,3 +40,29 @@ of Globalization Pipeline service to a specific file format, adding a new target
 language in bundle configuration, creating a new user and so on.
 
 To see available commands and options, please see [Globalization Pipeline Command Line Interface Guide](gp-cli.md)
+
+# Community
+
+* View or file GitHub [Issues](https://github.com/IBM-Bluemix/gp-java-tools/issues)
+* Connect with the open source community on [developerWorks Open](https://developer.ibm.com/open/ibm-bluemix-globalization-pipeline-service/python-sdk/)
```

Comment:
wrong link- should be https://developer.ibm.com/open/ibm-bluemix-globalization-pipeline-service/java-sdk/